src/ua_proxies.cpp

```diff
@@ -27,14 +27,15 @@
 using namespace std;
 
 UA_NodeId* nodePairList_getTargetIdBySourceId(nodePairList pairList, UA_NodeId remoteId) {
-  UA_NodeId* local = nullptr;
+/*  UA_NodeId* local = nullptr;
```

Comment:
Can we remove this function at all?
It is not used....

Reply:
Currently there is no further use planned. I will update the PR and remove the function.